docs/UserGuide.md

```diff
@@ -1078,27 +1090,27 @@ A: HomeBoss only supports English characters.
-| Add     | `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`                      | `customer add --name Gabriel --phone 87654321 --email gabrielrocks@gmail.com --address RVRC Block B`        |
-| View    | `customer view CUSTOMER_ID`                                                                          | `customer view 1001`                                                                                        |
-| List    | `customer list`                                                                                      | `customer list`                                                                                             |
-| Find    | `customer find KEYWORD [MORE_KEYWORDS]`                                                              | `customer find Ibrahim Yu`                                                                                  |
-| Edit    | `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]` | `customer edit 1001 --name Gabriel -–phone 1234 5678 --email gabrielrocks@gmail.com --address RVRC Block B` |
-| Delete  | `customer delete CUSTOMER_ID`                                                                        | `customer delete 1001`                                                                                      |
+| Command | Format                                                                                               | Examples                                                                                              |
+|---------|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| Add     | `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`                      | `customer add --name Gabriel --phone 87654321 --email gabrielrocks@gmail.com --address RVRC Block B`  |
+| View    | `customer view CUSTOMER_ID`                                                                          | `customer view 1`                                                                                     |
+| List    | `customer list`                                                                                      | `customer list`                                                                                       |
+| Find    | `customer find KEYWORD [MORE_KEYWORDS]`                                                              | `customer find Ibrahim Yu`                                                                            |
+| Edit    | `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]` | `customer edit 1 --name Gabriel -–phone 1234 5678 --email gabrielrocks@gmail.com --address RVRC Block B` |
```

Comment:
Example for phone prefix, I think the dashes used are different

Reply:
done